test/testcafe/spec/Smoke_spec.js

```diff
@@ -0,0 +1,35 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import { checkConsoleMessages, renderWidget } from '../framework/shared';
+import xhrIdentifyWithPassword from '../../../playground/mocks/data/idp/idx/identify-with-password.json';
+
+const identifyWithPasswordMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword);
+
+const identifyRequestLogger = RequestLogger(/idx\/identify/, {
+  logRequestBody: true,
+  stringifyRequestBody: true,
+});
+
+fixture('Smoke Test')
+  .meta('v3', true); // NOTE: tells parity-v3 script to run this fixture
```

Comment:
TODO come up with better name for this metadata flag for easier filtering

Reply:
seems OK. we're going to eventually be able to remove this metadata for filtering once we have all the tests passing anyways, right?